### DISCUSSION TITLE
Is eslint-config-prettier No Longer Needed When Using ESLint and Prettier Together, Starting from ESLint 8.53.0?

### DISCUSSION THREAD

**Discussion by dev-boku** [eslint/eslint#17971] [Q&A]
Since formatting rules have been removed from ESLint starting with version `8.53.0`, is `eslint-config-prettier` no longer necessary when using ESLint and Prettier together? 

Additionally, if I want to display Prettier rules as errors, would I only need `eslint-plugin-prettier`? I noticed that `eslint-plugin-prettier` still lists `eslint-config-prettier` as a peer dependency, and there hasn't been much discussion about this, so I am seeking clarification.

**Comment by Rec0iL99** [comment]
Hi @dev-boku, you can still use formatting rules from [`ESLint Stylistic`](https://github.com/eslint-stylistic/eslint-stylistic) which are the same rules that were deprecated in ESLint core. If you are not using any formatting rules, then I think it's safe to remove `eslint-config-prettier`. Please refer [issue](https://github.com/prettier/eslint-config-prettier/issues/276) opened in the `eslint-config-prettier` repo. 

Note: formatting rules were not removed from the ESLint core, they were just deprecated in `8.53.0`.

**Comment by nzakas** [comment]
Also note that we don't maintain `eslint-plugin-prettier`, so the best way to get answers about that plugin is to contact the maintainers.
